<H1>EXPRS-METHOD</H1>
<H2>Table of Contents</H1>

<div class="ml-5">
	<h4><A href="#mgi">Michigan Genomics Initiative (MGI)</A></h4>
	<ul>
	<li><P><A href="#mgicohort">Cohort</A></P> </li>
	<li><P><A href="#mgigene">Genotype Data</A></P></li>
	<li><P><A href="#mgiphen">Phenome</A></P></li>
	</ul>
</div>

<div class="ml-5">
	<h4><P><A href="#ukb">UK Biobank cohort (UKB)</A></P></h4>
	<ul>
	<li><P><A href="#ukbcohort">Cohort</A></P></li>
	<li><P><A href="#ukbgene">Genotype Data</A></P></li>
	<li><P><A href="#ukbphen">UKB Phenome</A></P></li>
	</ul>
</div>

<h4><P><A href="#exdata">Exposure data</A></P></h4>
<h4><P><A href="#exgwas">Exposure GWAS Summary Statistics</A></P></h4>

<h4><P><A href="#statmet">Statistical Methods</A></P></h4>

<div class="ml-5">
	<h4><P><A href="#exprs">ExPRS Construction</A></P></h4>
	<ul>
	<li><P><A href="#prsct">  LD clumping and P-value Thresholding (C+T)</A></P></li >
	<li><P><A href="#lasso">  Lassosum</A></P></li>
	<li><P><A href="#dbslmm">  Deterministic Bayesian Sparse Linear Mixed Model (DBSLMM)</A></P></li>
	<li><P><A href="#prscs"> PRS-CS</A></P></li>
</div>

<div class="ml-5">
	<h4><P><A href="#exeval">ExPRS Evaluation</A></P></h4>
	<ul>
	<li><P><A href="#b_trait">Binary Traits</A></P></li>
	<li><P><A href="#c_trait">Continuous Traits</A> </P></li>
	</ul>
</div>

<h4><P><A href="#catalog">Online Visual Catalog: ExPRSweb</A></P></h4>
<h4><P><A href="#ack">Acknowledgement</A></P></h4>
<h4><P><A href="#ref">Method References</A></P></h4>

<br>

<h4><A name="mgi">Michigan Genomics Initiative (MGI)</A></h4>

<b><A name="mgicohort">MGI Cohort</A></b>

<p class="text-justify">Adult participants aged between 18 and 101 years at
enrollment were recruited through the Michigan Medicine health system
between 2012 and 2020. Participants
have consented to allow research on both their biospecimens and EHR data, as
well as linking their EHR data to national data sources such as medical and
pharmaceutical claims data. Participants were primarily recruited through the
MGI - Anesthesiology Collection Effort while awaiting a diagnostic or
interventional procedure either at a preoperative appointment or on the day of
their operative procedure at Michigan Medicine. Additional participants were
recruited through the Michigan Predictive Activity and Clinical Trajectories
(MIPACT) Study, the Mental Health BioBank (MHB2), and the Michigan Genomics
Initiative-Metabolism, Endocrinology, and Diabetes (MGI-MEND) Study. The
data used in this study included diagnoses coded with the Ninth and Tenth
Revision of the International Statistical Classification of Diseases (ICD9 and ICD10)
with clinical modifications (ICD9-CM and ICD10-CM), laboratory measurements,
anthropometrics (height, and body
mass index [BMI]), vitals (systolic and diastolic blood pressure [SBP and
DBP]), health behavior (alcohol amount, smoker and drinker), sex, precomputed
principal components (PCs), genotyping batch, recruitment study, and age. Data
were collected according to the Declaration of Helsinki principles. MGI study
participants’ consent forms and protocols were reviewed and approved by the University
of Michigan Medical School Institutional Review Board (IRB ID HUM00099605 and
HUM00155849). Opt-in written informed consent was obtained. Additional details about MGI can be found
online (see <a href="https://precisionhealth.umich.edu/our-research/michigangenomics/">https://precisionhealth.umich.edu/our-research/michigangenomics/</a>).</p>

<b><A name="mgigene">MGI Genotype Data </A></b>

<p class="text-justify">DNA from 56,984 blood samples was
genotyped on customized Illumina Infinium CoreExome-24 bead arrays and
subjected to various quality control filters, resulting in a set of 502,255
polymorphic variants. Principal components and ancestry were estimated by
projecting all genotyped samples into the space of the principal components of
the Human Genome Diversity Project reference panel using PLINK (938
individuals) [1, 2]. Pairwise kinship was assessed with the software KING [3],
and the software FastIndep was used to reduce the data to a maximal subset that
contained no pairs of individuals with 3rd-or closer degree relationship [4].
We removed participants without EHR data and participants not of recent
European descent from the analysis, resulting in a final sample of 46,782
unrelated subjects. Additional genotypes were obtained using the Haplotype
Reference Consortium reference panel of the Michigan Imputation Server [5] and
included over 24 million imputed variants with R<sup>2 </sup>above 0.3 and minor allele frequency (MAF) above 0.01%.</p>

<b><A name="mgiphen">MGI Phenome</A></b>

<p class="text-justify">The MGI phenome was based on 
ICD9-CM and ICD10-CM code data for 46,782 unrelated, genotyped individuals of
recent European ancestry. Longitudinal time-stamped diagnoses were recoded to
indicators for whether a patient ever had given a diagnosis code recorded by
Michigan Medicine. These ICD9-CM and ICD10-CM codes were aggregated to form up
to 1,814 PheCodes using the PheWAS R package (as described in detail elsewhere [6-8]).
To minimize differences in age and sex distributions, avoid extreme
case-control ratios, and reduce the computational burden, we matched up to 10
controls to each case using the R package “MatchIt” [9]. Nearest neighbor
matching was applied for age and the first four principal components of the
genotype data (PC1-4) using Mahalanobis distance with a caliper/width of 0.25
standard deviations. Exact matching was applied for sex and genotyping array. A
total of 1,685 case-control studies with &gt;50 cases were used for our
analyses of the MGI phenome.</p>


<h4><A name="ukb">UK Biobank cohort (UKB)</A></h4>

<b><A name="ukbcohort">UKB Cohort</A></b>

<p class="text-justify">UKB is a population-based cohort
collected from multiple sites across the United Kingdom and includes over
500,000 participants aged between 40 and 69 years when recruited in 2006–2010 [10].
The open-access UK Biobank data used in this study included genotypes, ICD9 and
ICD10 codes, biomarker data, anthropometrics, vitals, women’s health, health
behavior, inferred sex, inferred White British ancestry, kinship estimates down
to third degree, birthyear, genotype array, and precomputed principal
components of the genotypes. UK Biobank received ethical approval from the NHS
National Research Ethics Service North West (11/NW/0382).</p>

<b><A name="mgigene">UKB Genotype Data</A></b>

<p class="text-justify">We used the UK Biobank Imputed
Dataset (v3) and limited analyses to the documented 408,595 White British [11]
individuals and 47,836,001 variants with imputation information score &gt;= 0.3
and MAF &gt;= 0.01% of which 22,933,317
overlapped with the imputed MGI data (see above). Two random subsets of 5,000
and 10,000 unrelated, White British individuals were used for LD analyses of
UKB-based summary statistics. Genotyping, quality control, and imputation are
described in detail elsewhere [6].</p>


<b><A name="mgiphen">UKB Phenome</A></b>

<p class="text-justify">The UK Biobank phenome was based on
ICD9 and ICD10 code data of 408,595 White British [11], genotyped individuals
that were similarly aggregated to PheCodes as MGI (see above, also described
elsewhere [12]). In contrast to MGI, there were many pairwise relationships
reported for UKB participants. </p>

<p class="text-justify">To retain a larger effective sample
size for each phenotype, we first selected a maximal set of unrelated cases for
each phenotype (defined as no pairwise relationship of 3<sup>rd</sup> degree or
closer [4, 13]) before selecting a maximal set of unrelated controls unrelated
to these cases. Similar to MGI, we matched up to 10 controls to each case using
the R package “MatchIt” [9]. Nearest neighbor matching was applied for
birthyear (as proxy for age because age at diagnosis was not available to us)
and PC1-4 (Mahalanobis-metric matching; matching window caliper/width of 0.25
standard deviations), and exact matching was applied for sex and genotyping
array. A total of 1,419 matched case-control studies with &gt;50 cases each
were used for our analyses of the UK Biobank phenome. </p>

<h4><A name="exdata">Exposure data</A></h4>

<p class="text-justify">For a set of 21 continuous and 7
binary exposures for which we could find freely available and complete GWAS summary
statistics we extracted the corresponding EHR data. For the binary common disorder
(type 2 diabetes, hypertension, insomnia, and sleep apnea) we use the PheWAS
code-based definitions (see MGI and UKB phenome above). Survey based measures
with multiple responses per person (never/past/current alcohol use and smoking
status) were recoded to never / ever responses. Continuous exposures with
multiple measurements across multiple visits (e,g, BMI, biomarkers, or blood
pressure values) were harmonized as follows: we first removed outliers per
individual per trait by using the interquartile range (IQR) criterion. Next, we
aggregated multiple entries per individual by using the mean value across
multiple entries per person per trait as the value of the corresponding
continuous trait. For the UKB cohort, we calculated the estimated Glomerular
Filtration Rate (eGFR) on the natural scale using the harmonized serum
creatinine values (data field 30700), race and sex information and the Chronic
Kidney Disease Epidemiology Collaboration (CKD-EPI) equation [14].</p>


<h4><A name="exgwas">Exposure GWAS Summary Statistics</A></h4>

<p class="text-justify">For each of the 28 exposures, we
collected complete GWAS summary statistics from up to four different sources:
(1) catalogued GWAS of the NHGRI EBI GWAS Catalog[15] (2) GWAS from the FinnGen
Consortium; (3) published GWAS meta-analyses; and (4) publicly available GWAS
summary statistics of phenome x genome screening efforts of the UK Biobank data
(Lee and Neale Lab, <a href="https://www.leelabsg.org/resources">https://www.leelabsg.org/resources</a>
and <a href="https://github.com/Nealelab/UK_Biobank_GWAS">https://github.com/Nealelab/UK_Biobank_GWAS</a>).
We only included GWAS summary statistics of studies that analyzed broad
European ancestry to match the ancestry of discovery GWAS and target cohorts
(MGI and UKB).</p>

<p class="text-justify">If needed, we lifted over
coordinates of GWAS summary statistics to human genome assembly GRCh37
(LiftOver, UCSC Genome Browser Store, <a href="https://genome-store.ucsc.edu">https://genome-store.ucsc.edu</a>).
Entries with missing effect alleles, or effect sizes were excluded. If effect
allele frequency (EAF) was reported in the summary statistics, we also compared
EAF between the discovery GWAS and the target dataset (MGI and/or UKB). If the
proportion of likely flipped alleles (whose RAF deviated more than 15% between
the datasets) was above 40% we excluded the GWAS as source for PRS
construction. These chosen thresholds were subjective and based on clear
differentiation between correct and likely flipped alleles on the two
diagonals, as noted frequently in GWAS meta-analyses quality control
procedures.</p>


<h4><A name="statmet">Statistical Methods</A></h4>

<b><A name="exprs">ExPRS Construction</A></b>

<p class="text-justify"><span style='color:black;
background:white'>We constructed the PRS for an individual j in the form </span>PRS<i><sub>j</sub></i>=<span
style='font-size:12.0pt;font-family:"Arial",sans-serif;position:relative;
top:4.0pt'><img width=39 height=16 src="ExPRSweb_MethodsTab.fld/image001.png"></span> where
<i>i </i>indexes the included variants for that trait, weight <span
style='font-size:12.0pt;font-family:"Arial",sans-serif;position:relative;
top:3.0pt'><img width=10 height=14 src="ExPRSweb_MethodsTab.fld/image002.png"></span> is
the log odds ratios retrieved from the external GWAS summary statistics for
variant <i>i,</i> and <span
style='font-size:12.0pt;font-family:"Arial",sans-serif;position:relative;
top:4.0pt'><img width=15 height=16 src="ExPRSweb_MethodsTab.fld/image003.png"></span> is
a continuous version of the measured dosage data for the risk allele of variant
<i>i</i> in subject <i>j. </i>To construct a PRS, one must determine which
genetic loci to include in the PRS and their relative weights. We have obtained
GWAS summary statistics from several external sources, resulting in several
sets of weights for each trait of interest. For each set of weights obtained from GWAS summary statistics
from the above-mentioned sources and each trait, we generated for each exposure
GWAS up to 5 different PRSs reflecting the 5 implementations of 4 different PRS
methods: the C + T (both, best guess genotype [GT] and dosage [DS] version)[16-18], lassosum,
DBSLMM[20] and PRS-CS[21].</p>

<i><A name="prsct">LD clumping and P-value Thresholding (C+T)</A></i>

<p class="text-justify">We performed linkage disequilibrium (LD)
clumping/pruning of variants with p-values below 0.1 by using the imputed
allele dosages of 10,000 randomly selected samples and a pairwise correlation
cut-off at r<sup>2 </sup>&lt; 0.1 within 1Mb window. We constructed many
different PRS across a fine grid of p-value thresholds. The p-value threshold
with the highest pseudo-R<sup>2</sup> (binary trait) or highest R<sup>2</sup>
(continuous traits) (see <b>PRS Evaluation</b> below) was used to define the
optimized “Clumping and Thresholding (C &amp; T)” PRS. We applied two
approaches for LD clumping: C + T (GT) and C+T (DS). Specifically, the “C +T
(GT)” is implemented by plink-1.9 using the best guess genotypes (GT, imputed
genotype dosages are rounded to the next integer) for LD calculations, while
“C+T (DS)” is implemented in R and considers the uncertainty of imputed
genotypes by using the dosage data (DS).</p>

<i><A name="lasso">Lassosum</A></i>

<p class="text-justify">Lassosum obtains PRS weights by
applying elastic net penalization to GWAS summary statistics and incorporating
LD information from a reference panel. Here, we used 5,000 randomly selected,
unrelated samples as the LD reference panel. We applied a MAF filter of 1 %
and, in contrast to the previous two approaches, only included autosomal
variants that overlap between summary statistics, LD reference panel, and
target panel. Each “Lassosum” run resulted in up to 76 combinations of the
elastic net tuning parameters s and λ, and consequently, in 76 SNP sets with
corresponding weights used to construct. We then selected the PRS with the pseudo-R<sup>2</sup>
(binary trait) or highest R<sup>2</sup> (continuous traits) to define the
“Lassosum” PRS (see <b>PRS Evaluation</b> below).</p>

<i><A name="dbslmm">Deterministic Bayesian Sparse Linear Mixed Model (DBSLMM)</A></i>

<p class="text-justify">DBSLMM assumes that the true SNP
effect sizes derive from a mixture of normal distributions and relies on an
efficient deterministic search algorithm for statistical inference. DBSLMM
requires both GWAS summary statistics and LD information from a reference
panel. Specifically, DBSLMM first selects SNPs with large effect in a
deterministic fashion through the C+T procedure and then directly obtains both
large SNP effect sizes and small SNP effect sizes through analytic forms. Here,
we used 5,000 randomly selected unrelated samples as the LD reference panel. We
applied a MAF filter of 1% and only included autosomal variants that overlap
between summary statistics, LD reference panel, and target panel. Heritability
estimates obtained from gemma (see above-mentioned procedure) were used as the
input of DBSLMM. All other parameters we used are the default parameters in the
“DBSLMM” software. For example, we set the cutoff of SNPs clumping and pruning
to be r<sup>2</sup>&lt; 0.1 within a 1Mb window and <i>p</i>-value &lt; 1e-06,
respectively. Each DBSLMM run resulted in one SNP set with corresponding
weights to construct the PRS. We used the default version of DBSLMM which does
not require cross-validation and refer to the obtained PRS as “DBSLMM” PRS.</p>

<i><A name="prscs">PRS-CS</A></i>

<p class="text-justify">PRS-CS utilizes a Bayesian
regression framework and assumes a continuous shrinkage (CS) prior on the
effect sizes. Specifically, we applied the default “auto” version of PRS-CS
that obtain weights through Gibbs sampling algorithm. Here, PRS-CS-auto uses a precomputed
LD reference panel based on external European samples of the 1000 Genomes
Project (“EUR reference”) to construct a PRS. We applied a MAF filter of 1 %
and only included autosomal variants that overlap between summary statistics,
LD reference panel, and target panel. The obtained PRS is referred to as
“PRS-CS” PRS.</p>

<p class="text-justify">For each trait and
set of GWAS summary statistics, these approaches usually resulted in up to five
PRS. However, approaches that resulted in less than 5 weights / variants were
excluded. Using the R package “Rprs” (<a href="https://github.com/statgen/Rprs">https://github.com/statgen/Rprs</a>),
the value of each PRS was then calculated for each MGI participant and, if the
GWAS source did to the best of our knowledge did not include UKB samples, also
for each UKB participant. For comparability of association effect sizes
corresponding to the continuous PRS across exposures and PRS construction
methods, we centered PRS values in MGI and UKB to a mean of 0 and scaled them
to have a standard deviation of 1.</p>

<b><A name="exeval">ExPRS Evaluation</A></b>

<p class="text-justify">To assess the predictive performance
of these generated PRS, each PRS was assessed through cross-validation in
either the MGI cohort or the UKB cohort: we split the data corresponding to
each trait in training (50% of the samples with gender ratio unchanged) and
test set (50% of the samples with gender ratio unchanged). We used the training
set to determine the PRS tuning parameter(s) and used the testing set to obtain
performance metric for that PRS. </p>

<p class="text-justify">For the PRS evaluations, except for when computing the pseudo-R<sup>2
</sup>for binary traits (which is a measure of marginal association of the ExPRS
with the exposure)[22], we fit the following model for each PRS and exposure
adjusting for covariates:</p>

<p class="text-justify">g(E(Exposure |
PRS, Age, Sex, Array, PC)) =<span
style='font-size:12.0pt;font-family:"Arial",sans-serif;position:relative;
top:4.0pt'><img width=306 height=16 src="ExPRSweb_MethodsTab.fld/image004.png"></span>,                                                                                             
(1)</p>

<p class="text-justify">where the PCs were the first four
principal components obtained from the principal component analysis of the
genotyped GWAS markers, where “Age” was the age at last observed diagnosis in
MGI and birthyear in UKB and where “Array” represents the genotyping array.</p>

<i><A name="b_trait">Binary Traits</A></i>

<p class="text-justify">We used Nagelkerke’s pseudo-R<sup>2</sup>
to select the tuning parameters within the “C+T” and Lassosum construction
methods (P-value for “C+T” SNP sets; s and λ for Lassosum) and kept the PRS
with the highest pseudo-R<sup>2</sup> for further analyses. For each PRS
derived for each GWAS source/method combination, we assessed the following
performance measures relative to observed disease status in MGI and UKB: (1)
overall performance with Nagelkerke’s pseudo-R<sup>2</sup> using R packages
“rcompanion”, (2) accuracy with Brier score using R package “DescTools”; and
(3) ability to discriminate between cases and controls as measured by the area
under the covariate-adjusted receiver operating characteristic (AROC; semiparametric frequentist inference)
curve (denoted AAUC) using R package “ROCnReg”. Firth's bias reduction method
was used to resolve the problem of separation in logistic regression (R package
“brglm2”).</p>

<i><A name="c_trait">Continuous Traits</A></i>

<p class="text-justify">For the PRS evaluations of
continuous traits, we used R<sup>2</sup>
to select the tuning parameters within the “C+T” and Lassosum construction
methods (P-value for “C+T” SNP sets; s and λ for Lassosum) and kept the PRS
with the highest R<sup>2</sup> for further analyses. For each PRS derived for
each GWAS source/method combination, we assessed the prediction performance in
terms of R<sup>2</sup> in MGI and UKB.</p>

<p class="text-justify">Unless otherwise stated, analyses were performed using R 4.1.1.</p>

<h4><A name="catalog">Online Visual Catalog: ExPRSweb</A></h4>
  
<p class="text-justify">The online open access visual catalog <i>ExPRSweb</i> was implemented using Grails as previously described [8]. </p>

<h4><A name="ack">Acknowledgement</A></h4>

<p class="text-justify">The authors acknowledge the
Michigan Genomics Initiative participants, Precision Health at the University
of Michigan, and the University of Michigan Medical School Data Office for
Clinical and Translational Research, the University of Michigan Medical School
Central Biorepository, and the University of Michigan Advanced Genomics Core
for providing data storage, management, processing, and distribution services,
and the Center for Statistical Genetics in the Department of Biostatistics at
the School of Public Health for genotype data curation, imputation, and
management in support of the research reported in this publication.<br>
Part of this research has been conducted using both the UK Biobank Resource
under application number 24460 and using results and data generated by previous
researchers who have used the UK Biobank Resource.</p>
This material is based in part upon
work supported by the National Institutes of Health/NIH (NCI P30CA046592 [LGF,
BM]), by the University of Michigan (UM-Precision Health Investigators Award
U063790 [LGF, SP, YM, BM]), and by the National Science Foundation under grant
number DMS-1712933. Any opinions, findings, and conclusions or recommendations
expressed in this material are those of the author(s) and do not necessarily
reflect the views of the National Science Foundation. </p>

<h4><A name="ref">Method References</A></h4>

<p class="text-justify">
	<ol>
		<li>Wang, C., et al., <i>Ancestry estimation and control of population stratification for sequence-based association studies.</i> Nat Genet, 2014. <b>46</b>(4): p. 409-15.</li>
		<li>Li, J.Z., et al., <i>Worldwide human relationships inferred from genome-wide patterns of variation.</i> Science, 2008. <b>319</b>(5866): p. 1100-4.</li>
		<li>Manichaikul, A., et al., <i>Robust relationship inference in genome-wide association studies.</i> Bioinformatics, 2010. <b>26</b>(22): p. 2867-73.</li>
		<li>Abraham, K.J. and C. Diaz, <i>Identifying large sets of unrelated individuals and unrelated markers.</i> Source Code Biol Med, 2014. <b>9</b>(1): p. 6.</li>
		<li>McCarthy, S., et al., <i>A reference panel of 64,976 haplotypes for genotype imputation.</i> Nat Genet, 2016. <b>48</b>(10): p. 1279-83.</li>
		<li>Fritsche, L.G., et al., <i>Association of Polygenic Risk Scores for Multiple Cancers in a Phenome-wide Study: Results from The Michigan Genomics Initiative.</i> Am J Hum Genet, 2018. <b>102</b>(6): p. 1048-1061.</li>
		<li>Carroll, R.J., L. Bastarache, and J.C. Denny, <i>R PheWAS: data analysis and plotting tools for phenome-wide association studies in the R environment.</i> Bioinformatics, 2014. <b>30</b>(16): p. 2375-6.</li>
		<li>Fritsche, L.G., et al., <i>Cancer PRSweb: An Online Repository with Polygenic Risk Scores for Major Cancer Traits and Their Evaluation in Two Independent Biobanks.</i> Am J Hum Genet, 2020. <b>107</b>(5): p. 815-836.</li>
		<li>Ho, D.E., et al., <i>MatchIt: Nonparametric Preprocessing for Parametric Causal Inference.</i> Journal of Statistical Software, 2011. <b>42</b>(8): p. 1-28.</li>
		<li>Sudlow, C., et al., <i>UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age.</i> PLoS Med, 2015. <b>12</b>(3): p. e1001779.</li>
		<li>Bycroft, C., et al., <i>Genome-wide genetic data on ~500,000 UK Biobank participants.</i> bioRxiv, 2017.</li>
		<li>Zhou, W., et al., <i>Efficiently controlling for case-control imbalance and sample relatedness in large-scale genetic association studies.</i> Nat Genet, 2018. <b>50</b>(9): p. 1335-1341.</li>
		<li>Michailidou, K., et al., <i>Association analysis identifies 65 new breast cancer risk loci.</i> Nature, 2017. <b>551</b>(7678): p. 92-94.</li>
		<li>Levey, A.S., et al., <i>A new equation to estimate glomerular filtration rate.</i> Ann Intern Med, 2009. <b>150</b>(9): p. 604-12.</li>
		<li>Buniello, A., et al., <i>The NHGRI-EBI GWAS Catalog of published genome-wide association studies, targeted arrays and summary statistics 2019.</i> Nucleic acids research, 2019. <b>47</b>(D1): p. D1005-D1012.</li>
		<li>Wray, N.R., M.E. Goddard, and P.M. Visscher, <i>Prediction of individual genetic risk to disease from genome-wide association studies.</i> Genome Res, 2007. <b>17</b>(10): p. 1520-8.</li>
		<li>International Schizophrenia, C., et al., <i>Common polygenic variation contributes to risk of schizophrenia and bipolar disorder.</i> Nature, 2009. <b>460</b>(7256): p. 748-52.</li>
		<li>Euesden, J., C.M. Lewis, and P.F. O'Reilly, <i>PRSice: Polygenic Risk Score software.</i> Bioinformatics, 2015. <b>31</b>(9): p. 1466-8.</li>
		<li>Mak, T.S.H., et al., <i>Polygenic scores via penalized regression on summary statistics.</i> Genet Epidemiol, 2017. <b>41</b>(6): p. 469-480.</li>
		<li>Yang, S. and X. Zhou, <i>Accurate and Scalable Construction of Polygenic Scores in Large Biobank Data Sets.</i> Am J Hum Genet, 2020. <b>106</b>(5): p. 679-693.</li>
		<li>Ge, T., et al., <i>Polygenic prediction via Bayesian regression and continuous shrinkage priors.</i> Nat Commun, 2019. <b>10</b>(1): p. 1776.</li>
		<li>Nagelkerke, N.J., <i>A note on a general definition of the coefficient of determination.</i> Biometrika, 1991. <b>78</b>(3): p. 691-692.</li>
	</ol>
</p>
